onnxscript/backend/onnx_export.py

```diff
@@ -150,25 +150,34 @@ def _to_str(s):
     return s
 
 
-def _attribute_value(attr):
-    if attr.HasField("f"):
+def _attribute_value(attr: onnx.AttributeProto):
```

Comment:
Possible to reuse onnx.helper.get_attribute_value https://onnx.ai/onnx/api/helper.html#id2?

Reply:
Yes, that sounds good